Tests/StoreKitUnitTests/UnitTestsConfiguration.storekit

```diff
@@ -73,7 +73,7 @@
             {
               "displayPrice" : "20.15",
               "internalID" : "DE6B065C",
-              "numberOfPeriods" : 1,
+              "numberOfPeriods" : 2,
```

Comment:
Changed this to `2` so the test is more complete, otherwise everything would pass with a value of `1`.